mysite/polls/views.py

```diff
@@ -2,6 +2,7 @@
 from django.http import HttpResponse
 from .models import Question
 from django.http import Http404
+from django.template import loader
```

Comment:
Import statement got removed in my previous commit and had to add it along with this commit to avoid the ```loader undefined``` error in the app.